src/Contract/Wallet.purs

```diff
@@ -20,18 +20,18 @@ module Contract.Wallet
 import Prelude
 
 import Contract.Address (getWalletAddress, getWalletCollateral)
-import Contract.Monad (Contract, ContractEnv, wrapContract)
+import Contract.Monad (Contract)
 import Contract.Utxos (getWalletUtxos) as Contract.Utxos
 import Control.Monad.Reader (local)
-import Ctl.Internal.Deserialization.Keys (privateKeyFromBytes) as Deserialization.Keys
-import Ctl.Internal.QueryM
+import Control.Monad.Reader.Class (asks)
+import Ctl.Internal.Contract.Wallet
   ( getChangeAddress
-  , getNetworkId
   , getRewardAddresses
   , getUnusedAddresses
   , getWallet
   , signData
-  ) as QueryM
+  ) as Contract
```

Comment:
as before, `Contract.Wallet` or `Wallet` is the convention for the import unless this is done only for export.